internal/applications/user/dto/user_dto.go

```diff
@@ -0,0 +1,39 @@
+package dto
+
+type (
+	UserRequest struct {
+		RoleId   int32  `json:"role_id" validate:"required"`
+		Name     string `json:"name" validate:"required"`
+		Email    string `json:"email" validate:"required"`
+		Password string `json:"password" validate:"required"`
```

Comment:
```suggestion
		Email    string `json:"email" validate:"required"`
		Password string `json:"password" validate:"required"`
```
2 ini bisa di buat example custom validator utk password strength level dan email format

Reply:
done

Reply:
sudah ditambahkan untuk email & password